ftdc/parser/parser.go

```diff
@@ -959,13 +1017,87 @@ func LaunchREPL(ftdcFilepath string) {
 				// parseStringAsTime outputs an error message for us.
 				graphOptions.vertLinesAtSeconds = append(graphOptions.vertLinesAtSeconds, goTime.Unix())
 			}
-		case cmd == "refresh" || cmd == "r":
-			NolintPrintln("Refreshing graphs with new data")
+		case strings.HasPrefix(cmd, "select"):
+			withoutCmd := strings.TrimPrefix(cmd, "select")
+			pieces := strings.Split(withoutCmd, ",")
+			if len(pieces) == 0 {
+				break
+			}
+			additions := make([]string, 0)
+
+			// sorting it so that the additions output is alphabetical, but otherwise unnecessary
+			for _, nameFilePair := range sorted(gpw.metricFiles) {
+				metricName := nameFilePair.Key
+				for _, piece := range pieces {
+					trimmedPattern := strings.TrimSpace(piece)
+					matched, err := regexp.MatchString(trimmedPattern, metricName)
```

Comment:
Let's make this case insensitive (e.g: `"(i:pattern)"`):
```
MatchString(fmt.Sprintf("(i:%v)", trimmedPattern), metricName)
```

Reply:
done